pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -138,223 +138,223 @@
 <!ENTITY pbSearchResultPopup-addUrlToTwitter.accesskey     "T">
 <!ENTITY pbSearchResultPopup-addUrlToLinkedIn.label        "LinkedIn">
 <!ENTITY pbSearchResultPopup-addUrlToLinkedIn.accesskey    "L">
-<!ENTITY searchResultPopup-systemPhotoImportMac.label      "Add to iPhoto">
+<!ENTITY searchResultPopup-systemPhotoImportMac.label      "Adicionar ao iPhoto">
 <!ENTITY searchResultPopup-systemPhotoImportMac.accesskey  "h">
 
 <!-- inline edit subject bar -->
-<!ENTITY msgAnnotation.title                         "Edit">
-<!ENTITY saveMessageEditing.button.label             "Save">
+<!ENTITY msgAnnotation.title                         "Editar">
+<!ENTITY saveMessageEditing.button.label             "Salvar">
 <!ENTITY saveMessageEditing.button.accesskey         "S">
-<!ENTITY saveMessageEditingAsCopy.button.label       "Save As Copy">
+<!ENTITY saveMessageEditingAsCopy.button.label       "Salvar como cópia">
 <!ENTITY saveMessageEditingAsCopy.button.accesskey   "a">
-<!ENTITY cancelMessageEditing.button.label           "Cancel">
+<!ENTITY cancelMessageEditing.button.label           "Cancelar">
 <!ENTITY cancelMessageEditing.button.accesskey       "C">
 
 <!-- inline message editing toolbar -->
-<!ENTITY boldEditMsgToolbarCmd.tooltip               "Bold">
-<!ENTITY italicEditMsgToolbarCmd.tooltip             "Italic">
-<!ENTITY underlineEditMsgToolbarCmd.tooltip          "Underline">
-<!ENTITY strikethroughEditMsgToolbarCmd.tooltip      "Strikethrough">
-<!ENTITY cropTextEditMsgToolbarCmd.tooltip           "Crop Text">
-<!ENTITY insertLinkEditMsgToolbarCmd.tooltip         "Insert new link or edit selected link's properties">
-<!ENTITY insertListEditMsgToolbarCmd.tooltip         "Apply or remove bulleted or numbered list">
-<!ENTITY noneMsgToolbarCmd.label                     "None">
+<!ENTITY boldEditMsgToolbarCmd.tooltip               "Negrito">
+<!ENTITY italicEditMsgToolbarCmd.tooltip             "Itálico">
+<!ENTITY underlineEditMsgToolbarCmd.tooltip          "Sublinhado">
+<!ENTITY strikethroughEditMsgToolbarCmd.tooltip      "Tachado">
+<!ENTITY cropTextEditMsgToolbarCmd.tooltip           "Cortar texto">
+<!ENTITY insertLinkEditMsgToolbarCmd.tooltip         "Inserir novo link ou editar as propriedades do link selecionado">
+<!ENTITY insertListEditMsgToolbarCmd.tooltip         "Aplicar ou remover lista numerada ou com marcadores">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY insertListEditMsgToolbarCmd.tooltip         "Aplicar ou excluir lista numerada ou com marcadores">